### ISSUE TITLE
Access to the WebRTC Data Channel

### ISSUE THREAD

**Issue by aakashs** [twilio/video-quickstart-objc#60]
Hey, the launch blog post for Twilio Video (https://www.twilio.com/blog/2015/04/announcing-video-nt.html) mentions access to the WebRTC data channel. Is this exposed in the iOS SDK?

If not, any suggestions as to how to sync small amounts of data in real time between clients with Twilio? i.e. for something like what's mentioned in the blog post - "For example, a user can draw on top of the video they’re viewing, and the data for that can be transferred and shown immediately on the other side." ?

Thanks!

**Comment by ceaglest** [comment]
Hi @aakashs,

We don't currently support the WebRTC data channel, but are considering adding this feature after our first General Availability release. In the meantime you could use another signaling channel to exchange small pieces of data between Participants. Some examples are:

Twilio Sync - [in developer preview](https://www.twilio.com/sync).
Twilio Chat - Higher level than sync and focused around channels. Currently [in beta](https://www.twilio.com/chat).
Google Firebase - Specifically the [realtime data sync](https://firebase.google.com/docs/database/), and FCM features.

I'll keep this ticket open until we have a chance to add Data Channel support.

Thanks,
Chris Eagleston

**Comment by ceaglest** [comment]
Hi @aakashs,

We've recently released a new feature called Data Track which is based upon WebRTC Data Channels. Unlike a Data Channel, a Data Track is a unidirectional broadcast source which is shared with all of the Participants in a Room. Our [2.0.0-preview5](https://www.twilio.com/docs/api/video/changelogs/ios#200-preview5-october-23-2017) release has support for Data Tracks in Peer-to-Peer Rooms, while Group Rooms support is still in progress.

If you're exchanging small pieces of data between Participants this feature might fit the bill. We've updated our QS example to demonstrate location sharing between peers.

Best,
Chris Eagleston
